MSTC-DA-IICT/Hacktoberfest23-ReACTivate
# Dance Club Page Created

Description:
This pull request adds event data for the Dance Club, which can be used to display upcoming events for the Dance Club on the website.

Changes Made
- Added event data for the Dance Club, including event names, dates, times, image file names, organizers, and descriptions.

How to Test
1. Clone this branch and run the application.
2. Visit the Dance Club's events page to see the new events listed.

 @ManojDhundhalva    Please review and merge this pull request. Thank you!
Screenschots:
![Dance_Club1](https://github.com/MSTC-DA-IICT/Hacktoberfest23-ReACTivate/assets/119687173/8ba52add-0247-4eb0-a551-cdb643191bfd)
![Dance_Club2](https://github.com/MSTC-DA-IICT/Hacktoberfest23-ReACTivate/assets/119687173/4a329cd3-aa27-49e1-8e87-73ff93006f68)
